### PR TITLE
[Enhancement] Support Analyze to collect statistics of replace type aggregated columns

### LINKED ISSUE #12766: Miss column statistic for primary key table
### Steps to reproduce the behavior (Required)
1. Create a primary key table
2. Analyze table

### Expected behavior (Required)

All columns have statistic, such as min/max, ndv, histogram.

### Real behavior (Required)

Only key columns have statistic, value columns do not.

Cause value columns are treated as `REPLACE` aggregation type, and won't collect column statistics for it.

image

### StarRocks version (Required)
- 2.0, 2.1, 2.2, 2.3, 2.4

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #12766 

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
BE Build Success

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 11 / 11 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/ast/ShowAnalyzeStatusStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/ShowBasicStatsMetaStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticUtils.java|8|8|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/StatisticsCollectJobFactory.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowBasicStatsMetaStmt.java [hunk 1]
```diff
@@ -49,7 +50,7 @@ public static List<String> showBasicStatsMeta(BasicStatsMeta basicStatsMeta) thr
         }
         row.set(1, table.getName());
 
-        long totalCollectColumnsSize = table.getBaseSchema().stream().filter(column -> !column.isAggregated()).count();
+        long totalCollectColumnsSize = StatisticUtils.getCollectibleColumns(table).size();
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/sql/ast/ShowBasicStatsMetaStmt.java:53]: shouldn't collect all aggregate columns.

**Comment by wanpengfei-git** [comment]
BE Build Success

**Review by kangkaisen — APPROVED** [review]

**Review by packy92 — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Comment by dulong41** [comment]
run starrocks_admit_test

**Comment by dulong41** [comment]
run starrocks_admit_test

**Pull request merged**